packages/frontend-shared/src/components/Tooltip.vue

```diff
@@ -90,4 +132,42 @@ withDefaults(defineProps<{
   }
 }
 
+.v-popper__popper.v-popper--theme-menu {
```

Comment:
I did not see the mocks but do we need to do this? 
Overriding a third party library styles seems like it'll be very hard to maintain. We have no guarentee they won't change the classes in a minor version bump.

Reply:
@mike-plummer thoughts on this one?

Reply:
Unfortunately yes, if we want to change the default styles from `floating-vue` this is basically the approach we have to take. The library does have a [theme system ](https://floating-vue.starpad.dev/guide/themes.html#custom-theme) but it's really just a thin veneer to allow layering/sharing CSS defns - we're defining the `menu` theme here the same way we had already defined a `tooltip` theme just above this